extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmAlwaysEnabledProcessor.java

```diff
@@ -12,4 +15,9 @@ public FeatureBuildItem featureBuildItem() {
         return new FeatureBuildItem(Feature.HIBERNATE_ORM);
     }
 
+    @BuildStep(onlyIf = HibernateSpatialAvailable.class)
+    void registerHibernateSpatialLogFilter(BuildProducer<LogCleanupFilterBuildItem> filters) {
+        filters.produce(new LogCleanupFilterBuildItem("org.hibernate.spatial.integration.SpatialService",
+                "Hibernate Spatial integration enabled"));
+    }
```

Comment:
You already added this to `extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateLogFilterBuildStep.java`.

Reply:
Oh right, I had added it here but then figured that was the best place for it - removed.